app/views/shared/_choose_your_course.html.erb

```diff
@@ -0,0 +1,20 @@
+<section class='jumbotron'>
+  <h3>Choose your course</h3>
+  <p>Get to know our different types of workshops</p>
+
+  <section class="carousel slide" data-ride="carousel" id="postsCarousel">
+    <div class="row" align="center">
+      <% @workshops.each do |workshop| %>
+      <div class="col-md-4">
+        <div class="card-block">
+          <%= image_tag workshop.image, style: 'height:12em' %>
```

Comment:
don't use inline `style` attributes. We have CSS for that. Give the image_tag a `class: 'workshop-image'` and style that instead